Remove counterparty frontend parts

Raibo opened the pull request:
https://github.com/happy-travel/agent-app-project/issues/799

mingazhev reviewed (approved):
To be merged after code freeze

Raibo commented:
Reworked master registration (also reworked registration request, example in the issue comments)
Changes in model of mail template d-25ff2449ce9949e0aadb7bde35d6f2dc (masterAgentRegistrationTemplateId)
changed RegistrationDataForAdmin
PreferredCurrency is added to AgencyInfo
get counterparty/contract-file moved to agency/contract-file
